### PR TITLE
Add missing "SwaggerEndpointInfo"

### PR DESCRIPTION
**Pull request by Ryddag**
# Description

Add 13 optional field to "SwaggerEndpointInfo".

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/Core/TypeNameGenerator.cs [hunk 1]
```diff
@@ -1,15 +1,57 @@
 ﻿using NJsonSchema;
 using System;
 using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
 
 namespace Nancy.Metadata.Swagger.Core
 {
+    //https://stackoverflow.com/a/41961738
```
Comment (Jaxelr) [inline src/Core/TypeNameGenerator.cs:9]: Could you move this to a new class called Type Translator?

File: src/Fluent/SwaggerEndpointInfoExtensions.cs [hunk 2]
```diff
@@ -139,6 +139,12 @@ public static SwaggerEndpointInfo WithSummary(this SwaggerEndpointInfo endpointI
             return endpointInfo;
         }
 
+        public static SwaggerEndpointInfo WithTags(this SwaggerEndpointInfo endpointInfo, IEnumerable<string> tags)
+        {
+            endpointInfo.Tags = tags.ToArray();
```
Comment (Jaxelr) [inline src/Fluent/SwaggerEndpointInfoExtensions.cs:142]: Could you add an elvis operator?

**Review by Jaxelr — CHANGES_REQUESTED** [review]
Thanks for the PR, it looks good for the most part, just some minor Request for changes

File: src/Core/TypeNameGenerator.cs (hunk 1 above)
Comment (Ryddag) [reply src/Core/TypeNameGenerator.cs:9]: I will look once i get time and get back to you

**Review by Jaxelr — APPROVED** [review]

**Comment by Jaxelr** [comment]
I made the changes requested locally, thanks for the help!

**Pull request merged**